### PR TITLE
Feature/issue 13 routing connexion

### LINKED ISSUE #13: Routing
**Priority :**
1

**Duration :** 
0.5

**Description:**
Connection route configuration

**File added / modified:**

**Additional context**

Add any other context or screenshots here.

### PR DESCRIPTION
**Pull request by vanmarcke**
Issue [#13]

Connection route configuration

### CODE REVIEW

File: src/Controller/SecurityController.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+use Symfony\Component\Security\Http\Authentication\AuthenticationUtils;
+
+class SecurityController extends AbstractController
+{
+    /**
+     * @Route("/login", name="app_login")
+     */
+    public function login(AuthenticationUtils $authenticationUtils): Response
+    {
+        // if ($this->getUser()) {
```
Comment (antoinetrouve) [inline src/Controller/SecurityController.php:17]: to remove

File: src/Controller/SecurityController.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+        return $this->render('security/login.html.twig', ['last_username' => $lastUsername, 'error' => $error]);
+    }
+
+    /**
+     * @Route("/logout", name="app_logout")
+     */
+    public function logout(): void
+    {
+        throw new \LogicException('This method can be blank - it will be intercepted by the logout key on your firewall.');
```
Comment (antoinetrouve) [inline src/Controller/SecurityController.php:34]: `use  LogicException` instead of `\LogicException`

File: src/Entity/SnoUser.php [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+namespace App\Entity;
+
+use App\Repository\SnoUserRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+#[ORM\Entity(repositoryClass: SnoUserRepository::class)]
+class SnoUser implements UserInterface, PasswordAuthenticatedUserInterface
```
Comment (antoinetrouve) [inline src/Entity/SnoUser.php:11]:
```suggestion
class SnowUser implements UserInterface, PasswordAuthenticatedUserInterface
```

**Review by antoinetrouve — APPROVED** [review]

File: src/Entity/SnoUser.php (hunk 3 above)
Comment (vanmarcke) [reply src/Entity/SnoUser.php:11]: This is a prefix, in my web host I only have one database, so I must prefix all my entities to be able to have several sites in my database

File: src/Controller/SecurityController.php (hunk 1 above)
Comment (vanmarcke) [reply src/Controller/SecurityController.php:17]: This function is uncommented in issue 14

File: src/Controller/SecurityController.php (hunk 2 above)
Comment (vanmarcke) [reply src/Controller/SecurityController.php:34]: I did not find the USE that corresponds

**Pull request merged**